core/dbt/contracts/graph/unparsed.py

```diff
@@ -452,12 +452,13 @@ class UnparsedMetric(dbtClassMixin, Replaceable):
     name: str
     label: str
     type: str
+    timestamp: str
```

Comment:
Why is this no longer optional?

Reply:
I'm honestly not sure why it was listed as optional in the past. This is a required field for all metric types so I changed to reflect that 😕